app/views/stats/subscriptions.html.erb

```diff
@@ -4,4 +4,28 @@
 </div>
 <div class="col-md-8 col-md-offset-1">
     <h2>Subscriptions</h2>
-</div>
+ 	<table class="table inline-grid">
+		<tr>
+			<thead>
+				<th>Subscriptions</th>
+				<th>Tags</th>
+			</thead>
+		</tr>
+		<% @tags.each do |map| %>
```

Comment:
Here we could rename `map` to `group`, since each item in `@tags` represents a group of tags.

Reply:
```suggestion
		<% @tags.each do |range, tags| %>
```

We could do this because the key will represent the range of numbers, and the value will contain the tags themselves, for example `['tagname', 46]`.

Reply:
oh alright. so I'll do away with the joining tags to the count with hyphens (-)